Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Use NVIDIA CUDA as base image
+FROM nvidia/cuda:12.4.1-devel-ubuntu22.04
```

Comment:
CUDA 12.4.1 requires quite a new NVIDIA Driver, >=550.54.15. At least it has to be mentioned in the README. It's quite a frequent failure point. We can also broaden driver support by using older CUDA versions like 12.1.

Reply:
Changed to cuda 12.1